dev-python/colorcet/colorcet-2.0.6.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..9} pypy3 )
+
+inherit distutils-r1
+
+DESCRIPTION="Collection of perceptually uniform colormaps"
+HOMEPAGE="
+	https://pypi.org/project/colorcet
+	https://github.com/holoviz/colorcet"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+BDEPEND=">=dev-python/setuptools-30.3.0[${PYTHON_USEDEP}]"
+DEPEND="
```

Comment:
Are these needed at build time? (Sometimes the answer is yes, but it's often no if it's pure Python)

Reply:
Sadly, yes. `setup.py` for HoloViz packages is more convoluted than expected (or needed, frankly). [The Colorcet `setup.py` contains this snippet](https://github.com/holoviz/colorcet/blob/ae9deb9e27cb73cc274ce15fa1a9ec8aa8b557dd/setup.py#L16), for example:

```
import pyct.build

def get_setup_version(reponame):
    ...
    try:
        from param import version
    except:
        version = None
    if version is not None:
        return version.Version.setup_version(basepath, reponame, archive_commit="$Format:%h$")
    else:
        print("WARNING: param>=1.6.0 unavailable. If you are installing a package, this warning can safely be ignored. If you are creating a package or otherwise operating in a git repository, you should install param>=1.6.0.")
```

We could probably patch out the `get_setup_version()` call and thus the install-time `param` import, but... the `pyct.build` system might dynamically hook into that import somehow.

Their build workflow looks suspiciously fragile to me, so it might be better to just leave this as is.